pyblish_qml/host.py

```diff
@@ -158,6 +162,32 @@ def validate():
             # The running instance has already been closed.
             _state.pop("currentServer")
 
+            
+def hide():
+    # get existing GUI
+    if _state.get("currentServer"):
```

Comment:
Should we throw an error here, if there isn't a `currentServer`, to let the caller know it didn't work? At the moment, it looks like it would silently fail.

Reply:
I think we could let this slide for now, because these two methods `host.validate` and `host.publish` in above lines work in the same fashion ( no error if no `currentServer`).
Should be in another PR to resolve them all together.

Reply:
I've essentially duplicated the code from `host.publish`. I've not thought deeply into how it works or how it doesn't work.
Since validate, publish, hide, quit (and possibly others later) share most of the code here, it would be better to create a decorator.
```
from functools import wraps
...

def proxy_call(func):
    @wraps(func)
    def proxy_call_wrapper(*args, **kwargs):
        # get existing GUI
        if _state.get("currentServer"):
            server = _state["currentServer"]
            proxy = Proxy(server)
            try:
                return func(proxy, *args, **kwargs)
            except IOError:
                # The running instance has already been closed.
                _state.pop("currentServer")
    return proxy_call_wrapper
...

@proxy_call
def hide(proxy):
    proxy.hide()
```
Or something like that. Error handling can be added once it is decided what kind should be there of course.

Reply:
Yes, I think that makes sense. Happy with that.